docs/core.adoc

```diff
@@ -10,22 +10,60 @@ It's main features are:
 
 == Installation
 
-Use the Cordova CLI to add the Core module as a plugin to your project:
+The `@aerogear/core` module is available via NPM:
 
 ----
-cordova plugin add @aerogearservices/core-cordova
+npm install @aerogear/core
```

Comment:
`--save` :P

Reply:
Depending on the version of NPM save will be default option.

Reply:
That's an even better reason to add it.

Reply:
`--save` has been on by default since npm 5, and latest nodejs LTS (carbon) uses 5.6: https://nodejs.org/en/download/releases/.

Current `npm help install` doesn't even list `--save` as a valid flag anymore, just the different flavors (`--save-dev, --no-save, etc`) so my vote is to keep it out of new documentation.